### ISSUE TITLE
Alt text for images in exhibits

### ISSUE THREAD

**Issue by doana** [omeka/plugin-ExhibitBuilder#132]
Hello! 

I'm wondering if you'll consider a change to the alt text that is used for files embedded in exhibits. 

## The Problem
Currently, the alt text is set to the title of the item that the file belongs to. This creates a situation where embedding multiple files from a single item results in multiple different images with the same alt text. This isn't appropriate since the alt text should describe the content of the image itself, and should be different for each file.

## Solutions
1. One potential solution would be to use the title of the file, rather than the title of the image as the alt text (a one-line change). This would make the exhibit builder plugin consistent with the way that alt text is handled in the core Omeka project. This would be my preferred option as we already set the titles for files as a matter of practice in order to have proper alt text on the core item pages. I'm happy to submit a PR for this. 

2. Another solution would be to expose the ability to set the alt text in the UI when embedding the file within exhibit builder (similar to the way you can set the caption). Obviously, this would be a much bigger change and would need more development and testing. From the user's perspective this would also require users to set the alt text a second time in exhibit builder (in addition to setting the title of the file itself).

3. Some other option I haven't thought of? 

Thanks!
-Adam

**Comment by doana** [comment]
See change here for option number 1: https://github.com/doana/plugin-ExhibitBuilder/commit/dba285994398e0288f752aacd72353668fdd2c0c

**Comment by zerocrates** [comment]
I wonder if we should just take out the handling of "alt" specially from that function entirely. The core's function for displaying a file that's used there will automatically use the "display_title" for the file as the alt if nothing else is passed: meaning it will use a Dublin Core Title if one exists, or the file's original filename if it doesn't.

**Comment by doana** [comment]
That would work for me!
